files/stage-build-cpu-arch.tpl

```diff
@@ -0,0 +1,60 @@
+### This is the Terraform-generated dev-build.yml workflow for the         ###
```

Comment:
one little type-o in the comment here calling this 'dev-build' rather than 'stage-build'.